include/rnp.h

```diff
@@ -33,6 +33,7 @@
 
 #include <stddef.h>
 #include "packet.h"
+#include "../common/constants.h"
```

Comment:
yes, of course. the thing is it must build. you need to change include. that's a main include file, you can't expect anybody it will have a ../common/constant.h file in it's directory tree